hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -174,6 +174,12 @@ public static boolean compareRowLists(List<?> expected, List<?> actual) {
         return true;
     }
 
+    public static String getHadoopNonExistingPath() {
```

Comment:
simply `getNonExistingPath()` or even `nonExistingPath()` ?

Reply:
I used `hadoopNonExistingPath` - it's tailored for hadoop. For example, if there was `/` at the end, the tests would fail, even though the path is the same.